packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -339,6 +339,17 @@ export class MirrorNodeClient {
             requestId);
     }
 
+    public async getContractResultsLogsByNextLink(
+        link: string,
+        requestId?: string
+    ) {
+        const nextLink = link.split('/api/v1/')[1];
```

Comment:
It seems because `/api/v1/` is in the beginning js will populate the position 0 with a en empty string ```
```
"/api/v1/some/other/url".split('/api/v1/')
[ '', 'some/other/url' ]
```

We can use `.replace('/api/v1/', '')` instead

Reply:
Good point.

Reply:
Done